.github/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+<!-- Please see https://github.com/zulip/zulip-terminal#contributor-guidelines ! -->
+
+**What does this PR do?**
+
+Overall description goes here!
+
+- [ ] New Feature?
+- [ ] Fixes Bug? [Fixes <issue>; Partially <issue> | Fixes point <n> of <issue> | Resolves an issue that's not filed]
+- [ ] Internal Refactor?
+
+**Tested?** <!-- Fine to leave some of these unchecked! -->
+- [ ] Manually
+- [ ] Tests adapted (if necessary)
+- [ ] Tests added/changed for any new behavior
+- [ ] Passed linting & tests (overall PR)
+- [ ] Passed linting & tests (each commit)
+
+<!-- See https://github.com/zulip/zulip-terminal#commit-style -->
+**Commit flow** <!-- if more than one commit; add/delete/fill-in as appropriate -->
+* first commit doing some thing
+* maybe multiple commits doing similar things
+* ...
+
+**Notes/Doubts/Concerns/Questions** <!-- if any; add/delete/fill-in as appropriate -->
```

Comment:
This field is great. :+1: 
Can we simplify the title to just "Notes/Concerns"?
Doubts/Concerns/Questions sounds pretty similar to me.

Reply:
I agree 4 is too many. Maybe just "Notes" and push the possible content into the examples.

Of course the titles are editable/removable, but people may not think that, so simple titles may be easier as users are more likely to think to at least fill in their own content I hope :)